@udir-design/react/src/utilities/hooks/useTableOfContents/useTableOfContents.stories.tsx

```diff
@@ -18,6 +18,7 @@ const meta = preview.meta<
   useTableOfContentsProps
 >({
   title: 'Utilities/useTableOfContents',
+  tags: ['beta'],
```

Comment:
Denne skal vere i alpha 😊 

```suggestion
  tags: ['alpha'],
```